src/OpenTelemetry.AutoInstrumentation/Configuration/ConfigurationKeys.cs

```diff
@@ -96,6 +97,18 @@ public static class Traces
         /// Configuration key for legacy source names to be added to the tracer at the startup.
         /// </summary>
         public const string LegacySources = "OTEL_DOTNET_AUTO_LEGACY_SOURCES";
+
+        /// <summary>
+        /// Configuration keys for instrumentation options.
+        /// </summary>
+        public static class InstrumentationOptions
+        {
+            /// <summary>
+            /// Configuration key for GraphQL instrumentation to enable passing query as a document tag.
```

Comment:
nit: in OTel these are called attributes

```suggestion
            /// Configuration key for GraphQL instrumentation to enable passing query as a document attribute.
```